pkg/escape/strings.go

```diff
@@ -11,6 +11,9 @@ var (
 	}
 
 	codesStr = map[string]string{}
+
+	// escaper   = strings.NewReplacer(`,`, `\,`, `"`, `\"`, ` `, `\ `, `=`, `\=`)
```

Comment:
these are here for reference for now.....I will follow this up with another change using these benchmarks as a reference.